run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH=/app/usr/lib
```

Comment:
this is still wrong, /app/usr makes no sense, it should be moved to /app/lib, then you don't have to set this.